Makefile

```diff
@@ -1027,39 +1027,41 @@ election-results/historical-state-labels.geojson: geojson/albers/state-labels.ge
 		| ./flatten-geojson \
 		> $@
 
-election-results/historical-$(RACE_TYPE)-%.geojson: geojson/albers/%.geojson \
+election-results/110m/historical-$(RACE_TYPE)-%.geojson: geojson/albers/%.geojson \
         election-results/historical-$(RACE_TYPE).csv
+	mkdir -p $(dir $@)
 	cat geojson/albers/$*.geojson \
 		| node_modules/.bin/geojson-join \
 			--format=csv --againstField=id --geojsonField=id election-results/historical-$(RACE_TYPE).csv \
 		| ./add-density-property voteCount \
 		| ./flatten-geojson \
 		> $@
 
-election-results/historical-$(RACE_TYPE)-%-10m.geojson: geojson/albers/us-10m/%.geojson \
+election-results/10m/historical-$(RACE_TYPE)-%.geojson: geojson/albers/us-10m/%.geojson \
```

Comment:
Using different directories for 10m and 110m (I'm assuming that's the correct resolution anyway) files since I was blocked on specificity issues again.